frodo.js

```diff
@@ -4,70 +4,75 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?